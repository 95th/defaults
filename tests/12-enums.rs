use defaults::Defaults;

#[derive(Defaults)]
enum Foo {
    A,
    B,
}

fn main() {}
