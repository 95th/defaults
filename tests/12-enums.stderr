error: Explicit default is required for enums.
       help: Add `#[def = "..."]` to the enum definition
 --> tests/12-enums.rs:4:6
  |
4 | enum Foo {
  |      ^^^
